.ruby-version

```diff
@@ -1 +1 @@
-3.0.5
+3.0.4
```

Comment:
Please revert, ruby version change should not be part of this PR.

Reply:
Done. See https://github.com/rapid7/metasploit-framework/pull/17669/commits/b2b9a382f7b3ef6cbccedcee0a0172c98c509d01